### PR TITLE
fix: select ui fixed on mobile screen

### PR DESCRIPTION
**Pull request by Sagargajare**
## What does this PR do?

Fixes # (#1297)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Enhancement (non-breaking small changes to existing functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How should this be tested?

- [ ] Test A
- [ ] Test B

## Checklist

- I haven't read the [contributing guide](https://github.com/lensterxyz/lenster/blob/main/CONTRIBUTING.md)
- My code doesn't follow the style guidelines of this project
- I haven't performed a self-review of my own code and corrected any misspellings
- I haven't commented my code, particularly in hard-to-understand areas
- I haven't checked if my PR needs changes to the documentation
- I haven't checked if my changes generate no new warnings
- I haven't added tests that prove my fix is effective or that my feature works
- I haven't checked if new and existing unit tests pass locally with my changes

## Preview
On a mobile screen with a width of 375px
image

On a mobile screen with a width of 414px
image

On Desktop Screen
image

### CODE REVIEW

File: apps/web/src/components/Home/SeeThroughLens.tsx [hunk 1]
```diff
@@ -110,7 +110,7 @@ const SeeThroughLens: FC = () => {
       >
         <Menu.Items
           static
-          className="absolute w-64 right-0 z-[5] mt-1 bg-white rounded-xl border shadow-sm dark:bg-gray-900 focus:outline-none dark:border-gray-700/80"
+          className="absolute w-64 right-0 max-[412px]:left-0 z-[5] mt-1 bg-white rounded-xl border shadow-sm dark:bg-gray-900 focus:outline-none dark:border-gray-700/80"
```
Comment (bigint) [inline apps/web/src/components/Home/SeeThroughLens.tsx:113]: **nitpick:** we don't recommend to use hardcoded pixels 🙈

Comment (Sagargajare) [reply apps/web/src/components/Home/SeeThroughLens.tsx:113]: Can you suggest better approach? 

I will surely go with that way

Comment (Sagargajare) [reply apps/web/src/components/Home/SeeThroughLens.tsx:113]: I searched for better approach
And only found this. 


Screenshot_2022-12-06-20-24-53-276-edit_com android chrome

**Comment by bigint** [comment]
This PR looks like have a better solution https://github.com/lensterxyz/lenster/pull/1365

So closing this 🙏🏼

**Pull request closed without merging**